.github/actions/repo-gardening/src/get-next-valid-milestone.js

```diff
@@ -29,7 +29,7 @@ async function getNextValidMilestone( octokit, owner, repo, plugin = 'jetpack' )
 	for await ( const response of responses ) {
 		// Find a milestone which name is a version number
 		// and it's due dates is earliest in a future
-		const reg = new RegExp( plugin + '/d.d' );
+		const reg = new RegExp( plugin + '\\/\\d\\.\\d' );
```

Comment:
Seems unrelated to this PR, but it is needed.
```suggestion
		const reg = new RegExp( '^' + plugin + '/\\d+\\.\\d+' );
```